catalog/camel-report-maven-plugin/src/main/java/org/apache/camel/maven/ValidateMojo.java

```diff
@@ -219,15 +218,13 @@ protected void doExecuteConfigurationFiles(CamelCatalog catalog) throws MojoExec
         // TODO: implement me
 
         Set<File> propertiesFiles = new LinkedHashSet<>();
-        List list = project.getResources();
-        for (Object obj : list) {
-            Resource dir = (Resource) obj;
+        List<Resource> list = project.getResources();
+        for (Resource dir : list) {
```

Comment:
```suggestion
        for (Resource dir : project.getResources()) {
```
It could simplified like this also